app/uk/gov/hmrc/tradergoodsprofilesrouter/models/response/eis/ReviewReason.scala

```diff
@@ -41,7 +41,7 @@ object ReviewReason extends Enum[ReviewReason] with PlayEnum[ReviewReason] {
       )
   case object measure
       extends ReviewReason(
-        "The restrictions have changed or there may be new restrictions. You need to categorise the record."
+        "To The commodity code or restrictions have changed. You'll need to categorise the record."
```

Comment:
That sentence doesn't read quite right. Is the definitely what it should be.

There is also an extra capital letter on the first The